### PR TITLE
SUCCESSFULLY SOLVED THE PROJECT CHANGES REQUESTED TO DO .

### PR DESCRIPTION
**Pull request by dnyanesh99borse**
## 📋 Description

ACTUALLY I HAD SUCCESSFULLY COMMITED ALL THE CHANGES TWO DAY AGO WITH PROPERLY SOLVING THE ISSUE IN EFFECTIVE MANNER AND SUCCESSFULLY RAISED THE PULL REQUEST FOR THAT ISSUE INCLUDING:
PR FOR ISSUE: #219 
PR FOR ISSUE: #207 
PR FOR ISSUE: #209 

WHERE SOME OF THE RE CHANGES ARE REQUESTED BY  DEAR SIR HENCE I RESOLVED THIS ISSUE AS PER THE REQUEST AND HENCE TO COMMIT THAT CHANGES I AM RAISING THIS PULL REQUES. 
SO TO BE HONEST PLEASE REVIEW THE CHANGES AND ACCEPT THE CHANGES AND PLEASE RESPONSE AND ENCOURAGE ME TO CONTRIBUTE.

## 🔨 Changes Made

FIXES FOR ISSUE: #219 
FIXES FOR ISSUE: #207 
FIXES FOR ISSUE: #209 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [ *] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [ *] My code follows the style guidelines of this project.
- [ *] I have performed a self-review of my own code.
- [ *] I have commented my code, particularly in hard-to-understand areas.
- [ *] I have made corresponding changes to the documentation (if applicable).
- [ *] I have tested it locally and it works fine.
- [ *] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ *] Bug fix (non-breaking change which fixes an issue) 🐛
- [ *] New feature (non-breaking change which adds functionality) ✨
- [ *] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

- Fixes :
- FIXES FOR ISSUE: #219 
FIXES FOR ISSUE: #207 
FIXES FOR ISSUE: #209 

## 📸 Screenshots (if applicable)
image

image

image

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Comment by dnyanesh99borse** [comment]
SIR I HAD SUCCESSFULLY COMMITED THE CHANGES AS YOU REQUESTED . PLEASE REVIEW THE CHANGES AND PLEASE MERGE THE PR PLEASE AND BY RESPECING MY HARDWORK PLEASE MERGE MY  ALL THREE PR WITH RESPECTIVE LEVEL LABELS AND PLEASE ENCOURAGE ME TO KEEP HAPPY CONTRIUTING.. 

      I HOPE YOU WILL .., ~Thankyou @Harshdev098

**Comment by dnyanesh99borse** [comment]
PLEASE REVEIW THE CHANGES SIR. @Harshdev098

File: public/index.html [hunk 1]
```diff
@@ -308,139 +310,205 @@ <h2>Why Join Us</h2>
...
+            <p class="section-description">Our stakeholders are the driving force behind our success. Their dedication,
+                expertise, and vision have helped shape our journey.</p>
+            <div class="stakeholder-grid">
+                <!-- Research Paper 1 -->
+                <div class="research-paper-card">
+                    <div class="paper-image">
+                        <img src="" alt="Research Paper 1">
+                    </div>
+                    <div class="paper-info">
+                        <h3>Advancements in Artificial Intelligence</h3>
```
Comment (Harshdev098) [inline public/index.html:462]: it is not visible, change  the color of the texts/heading

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by dnyanesh99borse** [comment]
SIR I HAD FINALLY COMPLETED THID REQUEST ALSO . NOW PLEASE PLEASE PLEASE I AM REQUESTING YOU TO PLEASE REVIEW THE CHANGES AND MERGE IT PLEASE PLEASE SO LET ME MOVE AHEAD IN MY CONTRIBUTION PLEASE SIR TRY TO UNDESTAND AND LET ME GLADLY ENJOY THE MORE CONTRIBUTIONS.

WELL STILL IF YOU WANT ANY ADDITION FEEL FREE TO ASK . ~Thankyou.

**Pull request merged**